### PR TITLE
Filter for article tags in admin

### PR DESCRIPTION
**Pull request by actlikewill**
image

This filters for article tags in the admin.

### CODE REVIEW

**Comment by actlikewill** [comment]
@longhotsummer Not all liis have the tag in their db. Should we run a migration to add it?

File: peachjam/admin.py [hunk 1]
```diff
@@ -1188,6 +1188,12 @@ def new_init(self, *args, **kwargs):
 class ArticleForm(forms.ModelForm):
     body = forms.CharField(widget=CKEditorWidget())
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        article_tags = Taxonomy.objects.filter(name="Article tags").first()
```
Comment (longhotsummer) [inline peachjam/admin.py:1193]: There's a helper method on taxonomy that will do this for you

Comment (longhotsummer) [reply peachjam/admin.py:1193]: Sorry, it's on Article. And it'll create if necessary.

Comment (actlikewill) [reply peachjam/admin.py]: added

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**